src/funding/applepay/config.jsx

```diff
@@ -0,0 +1,59 @@
+/* @flow */
+/** @jsx node */
+
+import { PLATFORM } from '@paypal/sdk-constants/src';
+import { getUserAgent, isIOS, isSafari } from 'belter';
+import { ApplePayLogo, LOGO_COLOR } from '@paypal/sdk-logos/src';
+
+import { BUTTON_COLOR, BUTTON_LAYOUT } from '../../constants';
+import { DEFAULT_FUNDING_CONFIG, type FundingSourceConfig } from '../common';
+
+export function getApplePayConfig() : FundingSourceConfig {
+    return {
+        ...DEFAULT_FUNDING_CONFIG,
+
+        requiresPopupSupport:             false,
+        requiresSupportedNativeBrowser:   false,
+        shippingChange:                   true,
+
+        platforms: [
+            PLATFORM.DESKTOP,
+            PLATFORM.MOBILE
+        ],
+
+        layouts: [
+            BUTTON_LAYOUT.HORIZONTAL,
+            BUTTON_LAYOUT.VERTICAL
+        ],
+
+        Logo: ({ logoColor, optional }) => ApplePayLogo({ logoColor, optional }),
+
+        colors: [
+            BUTTON_COLOR.BLACK,
+            BUTTON_COLOR.WHITE
+        ],
+
+        logoColors:  {
+            [ BUTTON_COLOR.BLACK ]:  LOGO_COLOR.WHITE,
+            [ BUTTON_COLOR.WHITE ]:  LOGO_COLOR.BLACK
+        },
+
+        eligible: ({ fundingEligibility }) => {
+            const eligibility = fundingEligibility.card;
+            const branded = Boolean(eligibility && eligibility.branded);
+
+            const ua = getUserAgent();
+            const isIosSafari = isIOS() && isSafari();
```

Comment:
Can't use these functions here, since this `eligible()` function will also be run on the server side without user agent. Anything we need, needs to be passed in as a prop. We can add an `applePaySupported` prop.

Reply:
Thinking we can abstract out an `isApplePaySupported` function to belter/device.js?